### ISSUE TITLE
Interactive Looper

### ISSUE THREAD

**Issue by fr3ddykrueger** [SimonThormeyer/beuthSoundGroup#70]
As a developer, I want to see looper-component to interact with the canvas, so that I can capture events.

duplicate of #9 

**description of the story**

The Looper can take events and/or collect the triggerd Events.

**blocked by**

**acceptance criteria**
- [ ] 

**tickets or tasks**

**story points**
